pages/landing.tsx

```diff
@@ -0,0 +1,277 @@
+import { useTranslation } from 'next-i18next'; // Translation hook
```

Comment:
We should rename this file to `pages/index.tsx` as it will be the applicant-facing homepage. The current `pages/index.tsx` should be moved to `pages/admin/index.tsx`.